crates/biome_graphql_formatter/src/prelude.rs

```diff
@@ -1,13 +1,9 @@
 //! This module provides important and useful traits to help to format tokens and nodes
 //! when implementing the [crate::FormatNodeRule] trait.
 
-#[allow(unused_imports)]
 pub(crate) use crate::{
     AsFormat, FormatNodeRule, FormattedIterExt as _, GraphqlFormatContext, GraphqlFormatter,
     IntoFormat,
 };
 pub(crate) use biome_formatter::prelude::*;
-#[allow(unused_imports)]
-pub(crate) use biome_rowan::{
-    AstNode as _, AstNodeList as _, AstNodeSlotMap as _, AstSeparatedList as _,
-};
```

Comment:
The `allow`s here in these preludes could make sense to keep for better DX when working on formatters.